src/bridge-did/src/op_id.rs

```diff
@@ -0,0 +1,58 @@
+use core::fmt;
+use std::borrow::Cow;
+
+use candid::CandidType;
+use ic_stable_structures::{Bound, Storable};
+use serde::{Deserialize, Serialize};
+
+/// Unique ID of an operation.
+#[derive(
+    Debug,
+    Default,
+    Copy,
+    Clone,
+    PartialEq,
+    Eq,
+    Ord,
+    PartialOrd,
+    CandidType,
+    Deserialize,
+    Serialize,
+    Hash,
+)]
+pub struct OperationId(u64);
+
+impl OperationId {
+    /// Creates new Id from the given number.
+    pub fn new(id: u64) -> Self {
+        Self(id)
+    }
+
+    /// Returns next OperationId.
+    pub fn next(&self) -> Self {
```

Comment:
I feel like this method can be misleading. Usually, `next` method on an id type means "get next unique id", but here it has a different meaning. Let's have the `next_id` method only where the actual id counter is stored.

Reply:
Fixed